plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/ConfluentSchemaRegistryConfig.java

```diff
@@ -40,6 +41,8 @@
     private int confluentSchemaRegistryClientCacheSize = 1000;
     private EmptyFieldStrategy emptyFieldStrategy = IGNORE;
     private Duration confluentSubjectsCacheRefreshInterval = new Duration(1, SECONDS);
+    private int confluentDescriptorCacheSize = 1000;
+    private Duration confluentDescriptorCacheRefreshInterval = new Duration(5, MINUTES);
```

Comment:
hmmm, IMO the cache shouldn't refresh automatically by default

Reply:
Updated to remove a configurable cache and instead use a non-evictable cache with a max size of 1000.